### DISCUSSION TITLE
JS formatter: `operator-linebreak`

### DISCUSSION THREAD

**Discussion by J-Sek** [biomejs/biome#3867] [Ideas]
It is a nobrainer for me so I will refrain from explanation. The reason Prettier had to be suppressed by ESLint in order to get this behavior just baffles me.

[Playground](https://biomejs.dev/playground/?lineWidth=120&indentStyle=space&quoteStyle=single&code=YwBsAGEAcwBzACAAQQAgAHsACgAgACAAcAB1AGIAbABpAGMAIABmAGkAZQBsAGQAIAA9ACAAXwBjAC4AQQBnAGcAcgBlAGcAYQB0AGkAbwBuAC4AUgBlAG4AdABhAGwAUgBhAHQAZQAKACAAIAAgACAAKwAgAF8AYwAuAEEAZwBnAHIAZQBnAGEAdABpAG8AbgAuAEwAZQBzAHMAUgBlAG4AdABhAGwAQQBiAGEAdABlAG0AZQBuAHQACgAgACAAIAAgACsAIABfAGMALgBBAGcAZwByAGUAZwBhAHQAaQBvAG4ALgBPAHAARQB4AAoAIAAgACAAIAAtACAAXwBjAC4AQQBnAGcAcgBlAGcAYQB0AGkAbwBuAC4ATABlAHMAcwBPAHAARQB4AEIAYQBzAGUACgAgACAAIAAgACsAIABfAGMALgBBAGcAZwByAGUAZwBhAHQAaQBvAG4ALgBFAGwAZQBjAHQAcgBpAGMAaQB0AHkAOwAKAH0A)

ESLint » rules » [operator-linebreak](https://eslint.org/docs/latest/rules/operator-linebreak)

One more [example](https://stackoverflow.com/questions/61952529/eslint-and-prettier-conflict-on-operator-linebreak-rule) of code demonstrating the issue.

Lack of that behavior prevents me from fully committing to migration from Prettier/ESLint to Biome.

**Comment by Conaclos** [comment]
Duplicate of #1693

**Reply by J-Sek** [reply]
> Duplicate of #1693

I was searching for 20 minutes among issues and PRs and somehow managed to miss it xD. Thanks for linking the first one.
